apps/frontend/src/app/3_organisms/StakingHistoryFrame/components/StakingWithdraws/StakingWithdraws.constants.tsx

```diff
@@ -35,6 +35,17 @@ export const COLUMNS_CONFIG = [
     ),
     sortable: true,
   },
+  {
+    id: 'slashedAmount',
+    title: t(translations.stakingHistory.unstakingPenalty),
+    cellRenderer: (tx: StakingWithdrawItem) => (
+      <AmountRenderer
+        prefix="-"
```

Comment:
I don't think you should prefix everything with `-` as `-0 SOV` looks really weird.